recipes-devtools/deepstream/deepstream-tao-apps_3.0.bb

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "NVIDIA Train, Adapt, and Optimize (TAO) Toolkit sample applications"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=dbef1fb16cd9e5c5249a8e0c5e639fb0"
+
+DS_VERSION = "6.0"
+DEPENDS = " deepstream-${DS_VERSION}"
+
+SRC_URI = " \
+    git://github.com/NVIDIA-AI-IOT/deepstream_tao_apps.git;branch=release/tao3.0_ds6.0ga \
+    file://0001-Cross-build-fixups.patch \
+"
+SRCREV = "2022175c07b7c76437f1e7a10ae90eb8517baf73"
```

Comment:
It looks like they don't tag their releases, so could you add

`PV .= "+g${SRCPV}"`

to append the git hash to the version number?  That will help with rebuilds if the SRCREV gets bumped.

Reply:
done!